package.json

```diff
@@ -19,6 +19,7 @@
 		"test:playwright:default": "npx cross-env TEST_SUITE=@default playwright test -c tests/playwright/playwright.config.ts",
 		"test:playwright:reverse-columns": "npx cross-env TEST_SUITE=@reverse-columns playwright test -c tests/playwright/playwright.config.ts",
 		"test:playwright:nested-tabs": "npx cross-env TEST_SUITE=@nested-tabs playwright test -c tests/playwright/playwright.config.ts",
+	  	"test:playwright:nested-tabs-html": "npx cross-env TEST_SUITE=@nested-tabs-html playwright test -c tests/playwright/playwright.config.ts",
```

Comment:
@davseve 

Hi David, 

I have tried adding this playwright tag, but it doesn't seem to be running yet. How can I activate this tag?